README.md

```diff
@@ -44,3 +63,25 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running the App in Docker using Dockerfile
+ To build the image:
+
+1. To run the docker file in dev mode:	     
+docker build --target development --tag todo-app:dev .
+To run the docker file in dev mode:      
+docker run --env-file ./.env -p 5000:5000 todo-app:dev
+Then Navigate to localhost:5000 
+   
+2. To build the docker file in prod mode:    
+docker build --target production --tag todo-app:prod .
+   To run the docker file in prod mode:       
+docker run --env-file ./.env -p 5000:80 todo-app:prod
+Then navigate to http://0.0.0.0:5000/
+
+3. To test local setup on UNIX:
+docker run --env-file ./.env -p 5000:5000 --mount type=bind,source="$(pwd)",target=/app/ todo-app:dev
```

Comment:
I'd argue that this should replace the dev run command above (as you'll want to be able to write code in dev mode!)